Cartfile

```diff
@@ -1,2 +1,2 @@
-github "pinterest/PINRemoteImage" "3.0.0-beta.6"
+github "pinterest/PINRemoteImage" "master"
```

Comment:
Can this be set to a specific version? Is there a specific commit in PINRemoteImage master that's needed? If so, I can do a release to help correct it.